src/frontend/screens/Settings/ProjectSettings/YourTeam/ReviewInvitation.tsx

```diff
@@ -0,0 +1,110 @@
+import {StyleSheet, View} from 'react-native';
+import {Text} from '../../../../sharedComponents/Text';
+import {defineMessages, useIntl} from 'react-intl';
+import {NativeNavigationComponent} from '../../../../sharedTypes';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import React from 'react';
+import {WHITE} from '../../../../lib/styles';
+import {Button} from '../../../../sharedComponents/Button';
+import {DeviceNameWithIcon} from '../../../../sharedComponents/DeviceNameWithIcon';
+import {RoleWithIcon} from '../../../../sharedComponents/RoleWithIcon';
+import {useProject} from '../../../../hooks/server/projects';
+import {WaitingForInviteAccept} from './WaitingForInviteAccept';
+import {useBottomSheetModal} from '../../../../sharedComponents/BottomSheetModal';
+import {MEMBER_ROLE_ID} from './SelectInviteeRole';
+import {useQueryClient} from '@tanstack/react-query';
+
+const m = defineMessages({
+  title: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.ReviewInvitation.title',
+    defaultMessage: 'Review Invitation',
+  },
+  youAreInviting: {
+    id: 'screens.Setting.ProjectSettings.YourTeam.ReviewInvitation.youAreInviting',
+    defaultMessage: 'You are inviting:',
+  },
+  coordinator: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.ReviewInvitation.coordinator',
+    defaultMessage: 'Coordinator',
+  },
+  participant: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.ReviewInvitation.participant',
+    defaultMessage: 'Participant',
+  },
+  sendInvite: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.ReviewInvitation.sendInvite',
+    defaultMessage: 'Send Invite',
+  },
+});
+
+export const ReviewInvitation: NativeNavigationComponent<
+  'ReviewInvitation'
+> = ({route, navigation}) => {
+  const {formatMessage: t} = useIntl();
+  const {role, ...rest} = route.params;
+  const [inviteSent, setInviteSent] = React.useState(false);
+  const {openSheet, ...restBottomSheet} = useBottomSheetModal({
+    openOnMount: false,
+  });
+  const project = useProject();
+  const queryClient = useQueryClient();
+
+  function sendInvite() {
+    setInviteSent(true);
+    project.$member
+      .invite(rest.deviceId, {roleId: role})
+      .then(val => {
+        if (val == 'ACCEPT')
+          queryClient.invalidateQueries({queryKey: ['projectMembers']});
+        navigation.navigate('InviteAccepted', {...route.params});
```

Comment:
Just realized there is no code path for when an invite is rejected, nor do we have a screen for that. Maybe a follow-up?

Reply:
Yeah, I discussed this with Sabella. We are just going to show the error screen with a message. Ill create a ticket and we can do it as follow up